entrypoint.sh

```diff
@@ -86,22 +86,28 @@ fi
 
 git add .
 
-if test -n "$(git status -s)"; then
-    git config user.name "$INPUT_GITCOMMITUSER"
-    git config user.email "$INPUT_GITCOMMITEMAIL"
-    git commit $INPUT_GITCOMMITFLAGS --message "$INPUT_GITCOMMITMESSAGE"
-
-    if test -n "$GH_PAGES_SSH_DEPLOY_KEY"; then
-        mkdir ~/.ssh
-        echo "$GH_PAGES_SSH_DEPLOY_KEY" > ~/.ssh/deploy_key
-        chmod 400 ~/.ssh/deploy_key
-
-        git remote rm origin
-        git remote add origin "git@github.com:$GITHUB_REPOSITORY.git"
+if test -z "$INPUT_NOCOMMIT"; then
```

Comment:
suggestion: this effectively says "if not no-commit", I think we should flip the conditional for better readability.

Reply:
Good point! Change made.